packaging/hudi-timeline-server-bundle/pom.xml

```diff
@@ -102,6 +102,10 @@
                     <groupId>javax.servlet</groupId>
                     <artifactId>*</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.eclipse.jetty</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
```

Comment:
Same here. Why not just exclude this while defining the dependency in the root POM instead of individually excluding it at all places ? This will not be manageable in the long run.

Reply:
ack